.chloggen/rm-dep-receiver-scrapererror.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: receiver/scraper
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: receiver/scraperhelper package is relocated to scraper/scraperhelper and receiver/scraperhelper marked deprecated.
```

nit: [Use active voice](https://developers.google.com/tech-writing/one/active-voice#prefer_active_voice_to_passive_voice) in changelogs
```suggestion
note: Move receiver/scraperhelper package to scraper/scraperhelper and deprecate original receiver/scraperhelper package.
```